scala-package/mxnet-demo/java-demo/Makefile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 SCALA_VERSION_PROFILE := 2.11
-MXNET_VERSION := 1.4.0-SNAPSHOT
+MXNET_VERSION := [1.4.0-SNAPSHOT,\)
```

Comment:
Can we change the minimum version to 1.5.0 ? Since the master is now at 1.5.0 and we are pushing out nightly snapshots with that version.

Reply:
Basically the version here would grab the Lastest package that have version 1.4.0-SNAPSHOT or above

Reply:
Ok!